### PR TITLE
dx-1008 removed payment_order variable from payment-url.md 

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/payment-url.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 {% assign when = include.when | default: "selecting the payment instrument" %}
 {% assign api_resource = include.api_resource %}
-{% assign payment_order = include.payment_order | default: false %}
 {% assign full_reference = include.full_reference | default: false %}
 
-{% if payment_order %}
+{% if api_resource == "checkout" %}
```
Comment (asbjornu) [inline _includes/payment-url.md:5]: `checkout` isn't an API resource. `checkout` needs to be changed to `paymentorders` in the entire file.

```suggestion
{% if api_resource == "paymentorders" %}
```

File: checkout/other-features.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ underlying payments or transaction.
 |                   | `callbackUrl`       | `string` | The URI to the API endpoint receiving `POST` requests on transaction activity related to the payment order.                                                                                                                                                                                              |
 |                   | `logoUrl`           | `string` | The URI to the logo that will be displayed on redirect pages. **HTTPS is a requirement**.                                                                                                                                                                                                                |
 
-{% include payment-url.md api_resource="checkout" payment_order=true
+{% include payment-url.md api_resource="checkout"
```
Comment (asbjornu) [inline checkout/other-features.md:62]:
```suggestion
{% include payment-url.md api_resource="paymentorders"
```

File: checkout/payment-menu.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ Remember to read up on our [URL resource][urls].
 
 {% include alert-risk-indicator.md %}
 
-{% include payment-url.md payment_order=true
+{% include payment-url.md api_resource="checkout"
```
Comment (asbjornu) [inline checkout/payment-menu.md:36]:
```suggestion
{% include payment-url.md api_resource="paymentorders"
```

File: _includes/payment-url.md [hunk 4]
```diff
@@ -59,8 +58,10 @@ you should also subscribe to other events provided by the Seamless View
 JavaScript and shut down the Seamless View if any of these events occur.
 
 Events to subscribe to for full control over the payment flow are can be found
-in {% if api_resource == "checkout" %} [Payment Menu Events](/{{ api_resource }}/other-features#payment-menu-events).
-{% else %} [Seamless View Events](/payments/{{ api_resource }}/other-features#seamless-view-events).
+in {% if api_resource == "paymentorders" %}
+{% assign url_resource= "checkout" %}[Payment Menu Events](/{{ url_resource}}/other-features#payment-menu-events).
```
Comment (asbjornu) [inline _includes/payment-url.md:62]: I didn't notice this URL before. We need to pass in `documentation_section` as well to this include and use that in the URL. `checkout` is going to generate wrong URLs for Payment Menu.

```suggestion
[Payment Menu Events](/{{ documentation_section }}/other-features#payment-menu-events).
```

File: _includes/payment-url.md [hunk 5]
```diff
@@ -59,8 +58,10 @@ you should also subscribe to other events provided by the Seamless View
...
+{% else %}
+[Seamless View Events](/payments/{{ api_resource }}/other-features#seamless-view-events).
```
Comment (asbjornu) [inline _includes/payment-url.md:64]:
```suggestion
[Seamless View Events](/payments/{{ documentation_section }}/other-features#seamless-view-events).
```

File: payment-menu/payment-order.md [hunk 6]
```diff
@@ -0,0 +1,313 @@
...
+var request = new XMLHttpRequest();
+request.addEventListener('load', function () {
+    // We will assume that our own backend returns the
+    // exact same as what SwedbankPay returns.
+    response = JSON.parse(this.responseText);
+    var script = document.createElement('script');
+    // This assumes the operations from the response of the POST of the
+    // payment order is returned verbatim from the server to the Ajax:
+    var operation = response.operations.find(function (o) {
+        return o.rel === 'view-consumer-identification';
```
Comment (payex-dev) [inline payment-menu/payment-order.md:108]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255&bid=18486443)

File: payment-menu/payment-order.md [hunk 7]
```diff
@@ -0,0 +1,313 @@
...
+    });
```
Comment (payex-dev) [inline payment-menu/payment-order.md:109]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md [hunk 8]
```diff
@@ -0,0 +1,313 @@
...
+        }
+    ]
+}
+```
+
+{:.table .table-striped}
+| Field          | Type     | Description                                                                        |
+| :------------- | :------- | :--------------------------------------------------------------------------------- |
+| `paymentorder` | `object` | The payment order object.                                                          |
+| └➔&nbsp;`id`   | `string` | {% include field-description-id.md resource="paymentorder" %}                      |
```
Comment (payex-dev) [inline payment-menu/payment-order.md:69]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md [hunk 9]
```diff
@@ -0,0 +1,313 @@
...
+
+## Step 2: Display the Payment Menu
+
+To load the payment menu from the JavaScript URL obtained in the back end API
+response, it needs to be set as a `script` element's `src` attribute. You can
+cause a page reload and do this with static HTML or you can avoid the page
+refresh by invoking the POST to create the payment order through Ajax and then
+create the script element with JavaScript, all inside the event handler for
+[`onConsumerIdentified`][technical-reference-onconsumer-identified].
+The HTML code will be unchanged in this example.
```
Comment (payex-dev) [inline payment-menu/payment-order.md:93]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md [hunk 10]
```diff
@@ -0,0 +1,313 @@
...
+            "contentType": "application/javascript"
+        }
+    ]
+}
+```
+
+{:.table .table-striped}
+| Field          | Type     | Description                                                                        |
+| :------------- | :------- | :--------------------------------------------------------------------------------- |
+| `paymentorder` | `object` | The payment order object.                                                          |
```
Comment (payex-dev) [inline payment-menu/payment-order.md:68]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md [hunk 11]
```diff
@@ -0,0 +1,313 @@
...
+To load the payment menu from the JavaScript URL obtained in the back end API
+response, it needs to be set as a `script` element's `src` attribute. You can
+cause a page reload and do this with static HTML or you can avoid the page
+refresh by invoking the POST to create the payment order through Ajax and then
+create the script element with JavaScript, all inside the event handler for
+[`onConsumerIdentified`][technical-reference-onconsumer-identified].
+The HTML code will be unchanged in this example.
+
+{:.code-header}
+**JavaScript**
```
Comment (payex-dev) [inline payment-menu/payment-order.md:96]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md [hunk 12]
```diff
@@ -0,0 +1,313 @@
...
+That script will then load the Seamless View for the Payment Menu. We will look
+into how to hook that up next.
+
+{% include alert.html type="informative" icon="info" body=" `orderReference` must
+be sent as a part of the `POST` request to `paymentorders` and must represent
+the order ID of the webshop or merchant website." %}
+
+## Step 2: Display the Payment Menu
+
+To load the payment menu from the JavaScript URL obtained in the back end API
```
Comment (payex-dev) [inline payment-menu/payment-order.md:87]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md [hunk 13]
```diff
@@ -0,0 +1,313 @@
...
+create the script element with JavaScript, all inside the event handler for
+[`onConsumerIdentified`][technical-reference-onconsumer-identified].
+The HTML code will be unchanged in this example.
+
+{:.code-header}
+**JavaScript**
+
+```js
+var request = new XMLHttpRequest();
+request.addEventListener('load', function () {
```
Comment (payex-dev) [inline payment-menu/payment-order.md:100]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255&bid=18486443)

File: payment-menu/payment-order.md (hunk 11 above)
Comment (payex-dev) [inline payment-menu/payment-order.md]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md (hunk 9 above)
Comment (payex-dev) [inline payment-menu/payment-order.md]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md (hunk 7 above)
Comment (payex-dev) [inline payment-menu/payment-order.md]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md (hunk 13 above)
Comment (payex-dev) [inline payment-menu/payment-order.md]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md (hunk 12 above)
Comment (payex-dev) [inline payment-menu/payment-order.md]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md (hunk 10 above)
Comment (payex-dev) [inline payment-menu/payment-order.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md (hunk 6 above)
Comment (payex-dev) [inline payment-menu/payment-order.md]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: payment-menu/payment-order.md (hunk 8 above)
Comment (payex-dev) [inline payment-menu/payment-order.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645255)

File: _includes/payment-url.md [hunk 14]
```diff
@@ -59,8 +59,10 @@ you should also subscribe to other events provided by the Seamless View
 JavaScript and shut down the Seamless View if any of these events occur.
 
 Events to subscribe to for full control over the payment flow are can be found
-in {% if api_resource == "checkout" %} [Payment Menu Events](/{{ api_resource }}/other-features#payment-menu-events).
-{% else %} [Seamless View Events](/payments/{{ api_resource }}/other-features#seamless-view-events).
+in {% if documentation_section == "payment-menu" or documentation_section=="checkout" %}
```
Comment (asbjornu) [inline _includes/payment-url.md:62]:
```suggestion
in {% if api_resource == "paymentorders" %}
```

File: _includes/payment-url.md [hunk 15]
```diff
@@ -59,10 +59,11 @@ you should also subscribe to other events provided by the Seamless View
...
 
 Events to subscribe to for full control over the payment flow are can be found
-in {% if documentation_section == "payment-menu" or documentation_section=="checkout" %}
-[Payment Menu Events](/{{ documentation_section}}/other-features#payment-menu-events).
+in {% if api_resource=="paymentorders" %} {% assign documentation_section="checkout" %}
+[Payment Menu Events](/{{ documentation_section
+}}/other-features#payment-menu-events).
 {% else %}
-[Seamless View Events](/payments/{{ documentation_section }}/other-features#seamless-view-events).
+[Seamless View Events](/payments/{{ api_resource }}/other-features#seamless-view-events).
```
Comment (asbjornu) [inline _includes/payment-url.md:66]:
```suggestion
[Seamless View Events](/payments/{{ documentation_section }}/other-features#seamless-view-events).
```

File: _includes/payment-url.md [hunk 16]
```diff
@@ -59,10 +59,11 @@ you should also subscribe to other events provided by the Seamless View
 JavaScript and shut down the Seamless View if any of these events occur.
 
 Events to subscribe to for full control over the payment flow are can be found
-in {% if documentation_section == "payment-menu" or documentation_section=="checkout" %}
-[Payment Menu Events](/{{ documentation_section}}/other-features#payment-menu-events).
+in {% if api_resource=="paymentorders" %} {% assign documentation_section="checkout" %}
```
Comment (asbjornu) [inline _includes/payment-url.md:62]:
```suggestion
in {% if api_resource=="paymentorders" %}
```

File: _includes/payment-url.md [hunk 17]
```diff
@@ -59,8 +59,11 @@ you should also subscribe to other events provided by the Seamless View
 JavaScript and shut down the Seamless View if any of these events occur.
 
 Events to subscribe to for full control over the payment flow are can be found
-in {% if api_resource == "checkout" %} [Payment Menu Events](/{{ api_resource }}/other-features#payment-menu-events).
-{% else %} [Seamless View Events](/payments/{{ api_resource }}/other-features#seamless-view-events).
+in {% if api_resource=="paymentorders" %}
+[Payment Menu Events](/{{ documentation_section
+}}/other-features#payment-menu-events).
```
Comment (asbjornu) [inline _includes/payment-url.md:64]:
```suggestion
[Payment Menu Events](/{{ documentation_section }}/other-features#payment-menu-events).
```

File: checkout/other-features.md [hunk 18]
```diff
@@ -59,7 +59,7 @@ underlying payments or transaction.
 |                   | `callbackUrl`       | `string` | The URI to the API endpoint receiving `POST` requests on transaction activity related to the payment order.                                                                                                                                                                                              |
 |                   | `logoUrl`           | `string` | The URI to the logo that will be displayed on redirect pages. **HTTPS is a requirement**.                                                                                                                                                                                                                |
 
-{% include payment-url.md api_resource="checkout" payment_order=true
+{% include payment-url.md api_resource="paymentorders"
```
Comment (asbjornu) [inline checkout/other-features.md:62]:
```suggestion
{% include payment-url.md api_resource="paymentorders" documentation_section="checkout"
```

File: checkout/payment-menu.md [hunk 19]
```diff
@@ -33,7 +33,7 @@ Remember to read up on our [URL resource][urls].
 
 {% include alert-risk-indicator.md %}
 
-{% include payment-url.md payment_order=true
+{% include payment-url.md api_resource="paymentorders"
```
Comment (asbjornu) [inline checkout/payment-menu.md:36]:
```suggestion
{% include payment-url.md api_resource="paymentorders" documentation_section="checkout"
```

File: _includes/payment-url.md [hunk 20]
```diff
@@ -59,8 +59,10 @@ you should also subscribe to other events provided by the Seamless View
 JavaScript and shut down the Seamless View if any of these events occur.
 
 Events to subscribe to for full control over the payment flow are can be found
-in {% if api_resource == "checkout" %} [Payment Menu Events](/{{ api_resource }}/other-features#payment-menu-events).
-{% else %} [Seamless View Events](/payments/{{ api_resource }}/other-features#seamless-view-events).
+in {% if api_resource=="paymentorders" %}
```
Comment (asbjornu) [inline _includes/payment-url.md:62]:
```suggestion
in {% if api_resource == "paymentorders" %}
```

**Pull request merged**